torch/nn/quantized/dynamic/modules/linear.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from ....modules.module import Module
+from ....modules.linear import Linear as NNLinear
+
+class Linear(NNLinear):
```

Comment:
can you please add setstate/getstate like other modules to make sure this module is serializable?

also please verify that it works with JIT (not sure whether weakscript annotation propagates to child classes)

Reply:
Will add the setstate/getstate like other modules for the serialization

Reply:
I currently adopted the approach of inheriting from `nnq.Linear`, so we now can just reuse the setstate/getstate from the base class. Of course a better way is to have the common base class. Will file a task to verify it works with JIT and check whether weakscript annotation propagates to child classes. For PyTorch 1.2 release, we currently only do the eager mode for the Linear module (this is not Script Module yet).